cmd/cni-metrics-helper/main.go

```diff
@@ -80,6 +81,16 @@ func main() {
 		}
 	}
 
+	pullIntervalEnv, found := os.LookupEnv("METRIC_PULL_INTERVAL")
```

Comment:
nit: sorry for going back on this, but I think the word "PULL" might be confusing to users. Can we do `METRIC_UPDATE_INTERVAL`? When we finally add a README for this helm chart, we can then make it clear that this is the rate at which metrics are updated, and they are published to CloudWatch at 2*update rate

Reply:
Completed in the latest commit